packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -203,31 +204,26 @@ export const VtexCommerce = (
     },
     session: (search: string): Promise<Session> => {
       const params = new URLSearchParams(search)
+      const authCookie = {
+        name: `VtexIdclientAutCookie_${account}`,
+        value:
+          getCookie(`VtexIdclientAutCookie_${account}`, ctx.headers.cookie) ??
+          null,
+      }
```

Comment:
Is all of this really necessary? Simply changing the request to POST doesn't solve the issue? From previous tests, I think it should.

Reply:
This was done to maintain  the session cookie without losing information from it. But this PR will be closed and I opened this [one](https://github.com/vtex/faststore/pull/1987) since we wont keep this logic :)